src/seedu/addressbook/AddressBook.java

```diff
@@ -1,4 +1,5 @@
 package seedu.addressbook;
+import java.util.*;
```

Comment:
You should import only what you need.  This reduces the chance of conflicting methods.